pinot-segment-local/src/main/java/org/apache/pinot/segment/local/startree/v2/builder/MultipleTreesBuilder.java

```diff
@@ -61,8 +62,10 @@ public class MultipleTreesBuilder implements Closeable {
   private final List<StarTreeV2BuilderConfig> _builderConfigs;
   private final BuildMode _buildMode;
   private final File _segmentDirectory;
+  private final File _indexDirectory;
```

Comment:
(minor) For convention, we usually call it `_indexDir` so that it is easier to map names